src/v0/destinations/google_adwords_offline_conversions/utils.js

```diff
@@ -63,6 +63,7 @@ const getConversionActionId = async (headers, params) => {
     let searchStreamResponse = await httpPOST(endpoint, data, requestOptions, {
       destType: 'google_adwords_offline_conversions',
       feature: 'transformation',
+      endpointPath: `/googleAds:searchStream`,
```

Comment:
Why not the entire endpoint? We can capture the version as well.

Reply:
We were taking just the ending path of the endpoint so as to avoid any PII data in form of `user-email` or `userID` to come up in the metrics, as some destinations have dynamic API endpoints based on these identifiers

Reply:
We can add a masking method to mask such PII data.

Reply:
@ItsSudip We thought of other approaches in the regard too, like placeholders /xx/xx/path/xx, but the label values needed to be as small as possible, and anyways, we can know what is the API in consideration by just the path hence made changes like this

Reply:
Got it